docs/extend/extend-cli/cli-devTutorials.md

```diff
@@ -1,74 +1,51 @@
-4. [Creating a new plug-in:](cli-developing-a-plugin.md) Create a new CLI plug-in that uses Zowe CLI programmatic APIs and a diff package to compare two data sets.
-5. [Implementing user profiles:](cli-implement-profiles.md) Implement user profiles with the plug-in.
+| General guidelines that apply to contributing to Zowe CLI and plug-ins | [Contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md) |
+| Conventions and best practices for creating packages and plug-ins for Zowe CLI | [Package and plug-in guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/PackagesAndPluginGuidelines.md)|
+| Guidelines for running tests on Zowe CLI | [Testing guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/TESTING.md) |
+| Guidelines for running tests on the plug-ins that you build| [Plug-in testing guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/PluginTESTINGGuidelines.md) |
+Versioning conventions for Zowe CLI and plug-ins| [Versioning guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/MaintainerVersioning.md) |
 
 ### Plug-in development overview
-At a high level, a plug-in must have `imperative-framework` configuration [(sample here)](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/pluginDef.ts).  This configuration is discovered by  `imperative-framework` through the [package.json](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/package.json) `imperative` key.
 
-A Zowe CLI plug-in will minimally contain the following:
+At a high level, a plug-in must have `imperative-framework` configuration (see a [sample here](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/pluginDef.ts)).  This configuration is discovered by  `imperative-framework` through the [package.json](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/package.json) `imperative` key.
+
+A Zowe CLI plug-in minimally contains the following:
 1. Programmatic API: Node.js programmatic APIs to be called by your handler or other Node.js applications.
 2. Command definition: The syntax definition for your command.
 3. Handler implementation: To invoke your programmatic API to display information in the format that you defined in the definition.
 
-The following guidelines and documentation will assist you during development:
-
-### Imperative CLI Framework documentation
-[Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) is a key source of information to learn about the features of Imperative CLI Framework (the code framework that you use to build plug-ins for Zowe CLI). Refer to these supplementary documents during development to learn about specific features such as:
+#### Imperative CLI Framework documentation
+**[this repo is archived. what else can we link to here? should we simply remove this section?]**
```

Comment:
@adam-wolfe please respond

Reply:
I've migrated the content to https://github.com/zowe/zowe-cli/wiki